packages/client-sdk-nodejs/src/internal/internal-auth-client.ts

```diff
@@ -105,16 +105,16 @@ export class InternalAuthClient implements IAuthClient {
       request.never = new Never();
     }
 
-    return await new Promise<GenerateAuthToken.Response>(resolve => {
+    return await new Promise<GenerateApiKey.Response>(resolve => {
       authClient.GenerateApiToken(
         request,
         {interceptors: this.interceptors},
         (err, resp) => {
           if (err || !resp) {
-            resolve(new GenerateAuthToken.Error(cacheServiceErrorMapper(err)));
+            resolve(new GenerateApiKey.Error(cacheServiceErrorMapper(err)));
```

Comment:
Do we need to update error messages?

Reply:
i didn't see any references to "token" in the error messages but let me know if you had anything else in mind I should be looking for!

Reply:
I wasn't sure if the errorMapper did any text translation. Looks like it doesn't. The server error messages might use auth token, but that is out of scope for this pr.